dashboard-api/internal/platform/server/handler/studies/read.go

```diff
@@ -0,0 +1,52 @@
+package studies
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	studiesmanager "github.com/vlab-research/vlab/dashboard-api/internal"
+)
+
+type readRequest struct {
+	Slug string `uri:"slug" binding:"required"`
+}
+
+type readResponse struct {
```

Comment:
Wait, this is the THIRD time I'm seeing the same type defined. I don't see how this is a good idea. Use an alias if you want abstraction, don't manually redefine it over and over!

Reply:
Thanks for the feedback, didn't know that go supported type aliases. 💯

Thinking more about this, for now it makes more sense to just return the entity. (and not using a type alias or extra type).

If in the future for some controller we need to return a stripped-down version of an entity, that would be a good case for an extra type (eg. User, PublicUser).